api/HTMLFormControlsCollection.json

```diff
@@ -5,10 +5,10 @@
         "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLFormControlsCollection",
         "support": {
           "chrome": {
-            "version_added": true
+            "version_added": "25"
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement#Browser_compatibility says `form.elements` was in Chrome 1, so this might be a case of interface object not being exposed. Can you check what kind of an object `form.elements` was in Chrome 24?

Reply:
I figured out the story for WebKit in https://github.com/mdn/browser-compat-data/pull/7758 which helps resolve this. It was added in https://trac.webkit.org/changeset/135093/webkit, which was [WebKit trunk 537.20.0](https://trac.webkit.org/browser/webkit/trunk/Source/WebCore/Configurations/Version.xcconfig?rev=135093) and that indeed is Chrome 25. So this was right!